README.md

```diff
@@ -45,12 +45,16 @@ Before proceeding with steps from this section complete steps listed in 'Prerequ
 1. Generate build system by invoking `./configure` from within WolkGateway base directory.
 Build system is generated to `out` directory
 2. Change current directory to `out`. Following steps are performed from within this directory
-3. Build WolkGateway by invoking `make all -j6`
-4. Run WolkGateway tests by invoking `tests`
+3. Build WolkGateway by invoking `make all -j$(nproc)`
+4. Run WolkGateway tests by invoking `make tests -j$(nproc)` and `./WolkGatewayTests`.
```

Comment:
OR

Reply:
It's not OR, you have to run `make tests -j$(nproc)` first always, as it build and then executes the tests. I'll clarify it now that you only need to execute the executable if you want to run the test again (but only after compiling them).